### DISCUSSION TITLE
Add playtime support for Ableton Push

### DISCUSSION THREAD

**Discussion by j0b333** [helgoboss/helgobox#1061] [Ideas]
The controller is literally made for this use case. Support would be amazing.

**Comment by acatpat** [comment]
Hi j0b333,
I manage to make it starting with the Preset von another Controller.
in the Controller Preset I save all the Pad, button and knob as virtual button for the Push 1 or 2.
It s really easy to do it.

I would be happy to get acces to the display. If someone has a Idee how...

**Reply by acatpat** [reply]
I think I took the preset novation lauchpad pro mk3 playtime for it

**Comment by acatpat** [comment]
![push 1   2 realearn playtime](https://github.com/user-attachments/assets/a7dff962-bf24-43bb-863c-f450929002ca)
